templates/control/edit_searchtagblacklist.html

```diff
@@ -0,0 +1,23 @@
+$def with (tags)
+$:{RENDER("common/stage_title.html", ["Edit Searchtag Blacklist", "Settings"])}
+
+<div id="usersearchtagblacklist-content" class="content form clear">
+  <form method="POST" class="form skinny clear" action="/control/editsearchtagblacklist">
+    $:{CSRF()}
+    <h3>Searchtag Blacklist Information</h3>
+    <p>
+      Tags you add here will not be able to be added to your submitted items by
```

Comment:
This might be too detailed. How about:

> Other users won’t be able to add tags listed here to your submissions. In addition to blocking individual tags, …

and drop the part about “must have at least two additional characters”.